core/src/test/java/org/elasticsearch/index/mapper/date/SimpleDateMappingTests.java

```diff
@@ -146,12 +149,25 @@ public void reset() {
     int i = 0;
 
     private DocumentMapper mapper(String type, String mapping) throws IOException {
+        return mapper(type, mapping, Version.CURRENT);
+    }
+
+    private DocumentMapper mapper(String type, String mapping, Version version) throws IOException {
         final String indexName = "test-" + (i++);
-        IndexService index = createIndex(indexName);
+        IndexService index;
+        if (version.equals(Version.CURRENT)) {
+            if (randomBoolean()) {
+                index = createIndex(indexName);
+            } else {
+                index = createIndex(indexName, settingsBuilder().put(IndexMetaData.SETTING_VERSION_CREATED, version).build());
```

Comment:
Do we really need this randomization? seems like the wrong place to test the version created setting is working properly.